packages/components/inputs/input-utils/src/styles.ts

```diff
@@ -42,29 +42,39 @@ const getInputFontColor = (props: TInputProps) => {
   return designTokens.fontColorForInput;
 };
 
+const getInputBorderWidth = (props: TInputProps) => {
+  if (props.hasError) {
+    return designTokens.borderWidthForInputWhenError;
+  }
+  if (props.hasWarning) {
+    return designTokens.borderWidthForInputWhenWarning;
+  }
+  return designTokens.borderWidthForInput;
+};
+
 const getInputStyles = (props: TInputProps) => {
   return css`
     appearance: none;
     background-color: ${props.isDisabled || props.disabled
       ? designTokens.backgroundColorForInputWhenDisabled
       : designTokens.backgroundColorForInput};
-    border: 1px solid ${getInputBorderColor(props)};
-    border-radius: ${designTokens.borderRadiusForInput};
+    border: ${getInputBorderWidth(props)} solid ${getInputBorderColor(props)};
+    border-radius: ${designTokens.borderRadius4};
     box-sizing: border-box;
     color: ${getInputFontColor(props)};
     cursor: ${props.isDisabled ? 'not-allowed' : 'default'};
     display: flex;
     flex: 1;
     font-family: inherit;
     font-size: ${designTokens.fontSizeForInput};
-    height: ${designTokens.sizeHeightInput};
-    min-height: ${designTokens.sizeHeightInput};
+    height: ${designTokens.heightForInput};
+    min-height: ${designTokens.heightForInput};
     opacity: ${props.isDisabled || props.disabled
       ? '1'
       : 'unset'}; /* fix for mobile safari */
     outline: none;
     overflow: hidden;
-    padding: 0 ${designTokens.spacing20};
+    padding: 0 ${designTokens.spacingM};
```

Comment:
I guess we need a new decision token here.
In the default theme we should keep using the `spacing20` (8px) choice token but in the new theme we need to use `spacing30` instead.

We could have a new decision token like this (in the `definitions.yaml` file):
```
// Default theme
paddingForInput: spacing-20

// New theme
paddingForInput: spacing-30

Reply:
Updated here [9bbdd7f](https://github.com/commercetools/ui-kit/pull/2347/commits/9bbdd7ff1f26ae4896b6164aeee65a48447b15c7). Thanks